mentat/config.py

```diff
@@ -72,7 +73,7 @@ class Config:
 
     # Context specific settings
     file_exclude_glob_list: list[str] = attr.field(
-        default=[],
+        default=cast(List[str], []),
```

Comment:
I fixed these errors in #282; instead of using default, I used factory (which you need for objects or attr's copies the reference to the empty list), and you can also just do list[str]() instead of casting an empty list.